manifests/010-prometheus-rules.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    name: samples-operator-alerts
+  name: samples-operator-alerts
+  namespace: openshift-cluster-samples-operator
+spec:
+  groups:
+  - name: SamplesOperator
```

Comment:
None of these alerts have a `for` condition, they shouldn't fire on a single sample having the respective value. I'd suggest at least `15m` or higher.

Reply:
just pushed `for` 's for each